tweakwcs/tpwcs.py

```diff
@@ -57,7 +57,13 @@ def _tp2tp(tpwcs1, tpwcs2, s=None):
     if 'fit_info' in tpwcs1.meta:
         center = np.array(tpwcs1.meta['fit_info']['center'])
     else:
-        center = np.zeros(2)
+        if tpwcs2.wcs.bounding_box is None:
+            # TODO: A pissible improvement would be to get an estimate
+            #       of "center" (where scale is estimated) from source
+            #       positions (if any).
+            center = np.zeros(2)
+        else:
+            center = np.mean(tpwcs2.wcs.bounding_box, axis=1)
```

Comment:
If you use wcs.pixel_bounds it should work for FITS and GWCS. `bounding_box` is gwcs specific.

Reply:
Ah. I keep getting confused with this.

Reply:
@nden Done. Thanks